build.gradle

```diff
@@ -54,14 +54,18 @@ compileJava {
     ]
 }
 
-ext.seleniumVersion = '[3.5.2,3.5.2]'
+ext.seleniumVersion = '[3.5.3,3.5.3]'
```

Comment:
what is the point to set a range, which includes only one item? Isn't it simply equal to '3.5.3'?

Reply:
@mykola-mokhnach The purpose is to make a valid pom.xml file. The version like 3.5+ is malformed for maven.